src/language/utils.ts

```diff
@@ -1,9 +1,9 @@
-export const hentPeriodeTittelMedTall = (
-  perioder: any[],
+export const hentTittelMedNr = (
+  nummer: any[],
```

Comment:
Skal det være number[]?

Reply:
Oops! Skulle ikke hete nr, men liste :) 
Kan være en liste av ulike typer. For utenlandsopphold vil det være IUtenlandsopphold, mens en liste med arbeidsgivere vil være av typen IArbeidsgiver 👩‍💻